pkg/controller/operatorsource/operatorsource_controller.go

```diff
@@ -33,7 +34,7 @@ func newReconciler(mgr manager.Manager) reconcile.Reconciler {
 }
 
 // add adds a new Controller to mgr with r as the reconcile.Reconciler
```

Comment:
This comment is out of sync with the function.

Reply:
This is also present in the `CatalogSourceConfig` code. I will fix it there too.